tasks/models/workflow.py

```diff
@@ -42,6 +42,8 @@ class TaskItem(QueueItem):
     """Task item queue management for Task instances (these should always be
     subtasks)."""
 
+    queue_field = "queue"
```

Comment:
Would it now make sense to name this something like `task_workflow` or just `workflow`?
```suggestion
    queue_field = "workflow"
```
and then
```
    queue = models.ForeignKey(
        TaskWorkflow,
        related_name="queue_items",
        on_delete=models.CASCADE,
    )
```
would become
```
    workflow = models.ForeignKey(
        TaskWorkflow,
        related_name="workflow_items",
        on_delete=models.CASCADE,
    )
```
It'd break a whole load of code, so could be pulled out into a separate PR.